doc/build-windows-mingw.md

```diff
@@ -29,7 +29,7 @@ First, install the general dependencies:
 
     sudo apt update
     sudo apt upgrade
-    sudo apt install build-essential libtool autotools-dev automake pkg-config bsdmainutils curl git
+    sudo apt install build-essential cmake pkg-config bsdmainutils curl git
```

Comment:
> If this list is still meant to be standalone everything needed for a win cross-compile on linux, then `libtool autotools-dev automake` shouldn't be removed, because they are needed.

You are right. Fixed.